### ISSUE TITLE
np.float is deprecated

### ISSUE THREAD

**Issue by am-ac** [ksmet1977/luxpy#28]
using newest numpy (1.24.2) with luxpy 1.9.8, both installed via pip, leads to an error using the spd_to_cri() function. In fact, np.float is apparently deprecated and that might lead to more issues. A grep search finds at least seven files using np.float.

**Comment by ksmet1977** [comment]
Thanks for reporting this. I've updated the version on github. A new release on pypi will come soon. (Btw, I've also changed np.int to np.int32 or int)

**Comment by Paul-Sims** [comment]
I installed luxpy in a fresh venv to troubleshoot why I am encountering this np.float error message.
I am using the default versions installed from using pip command in the instructions: 

luxpy version  v1.9.8
numpy version  1.25.1
Python version 3.10.5

So far, I have encountered np.float errors in colortransforms.py and helpers.py. I changed them in the local install for my project environment so I can move on with my code project.

Has the update you mentioned above been released on pypi?

Thanks for the great software package.

Paul
